R-package/R/lightgbm.R

```diff
@@ -189,7 +194,7 @@ lightgbm <- function(data,
   }
 
   # Set validation as oneself
-  if (verbose > 0L) {
+  if (params[["verbosity"]] > 0L) {
     train_args[["valids"]][["train"]] <- dtrain
   }
```

Comment:
Quite strange behavior, to be honest...
Maybe remove it in `v4.0.0`?

Reply:
I totally agree! I definitely think we should consider removing this prior to v4.0.0.

This has been in `{lightgbm}` since the very first version of the R package, in #168.

Wrote up a separate issue for discussion of that: #5203.

Reply:
Thanks a lot for doing this!